src/js/tests/task-completion-prompt.test.js

```diff
@@ -0,0 +1,44 @@
+/* Light tests for the task-completion prompt.
+ * (Note that coverage isn't total) */
```

Comment:
I could add more coverage (in addition to the manual QA i've done) but this ticket has gone on long enough - I may well add more tests after review, as a separate PR

Reply:
Yeah I think it'd be great if we could check whether the survey prompt is showing, instead of checking the cookie itself (which is more of an implementation detail).